examples/test-upgrade/v100/cloud-backup-policies/v091/main.tf

```diff
@@ -0,0 +1,69 @@
+// This will Create a Project,  Cluster and Modify the 4 Default Policies Simultaneously
+
+resource "mongodbatlas_project" "project_test" {
+  name   = var.project_name
+  org_id = var.org_id
+}
+
+resource "mongodbatlas_cluster" "cluster_test" {
+  project_id   = mongodbatlas_project.project_test.id
+  name         = var.cluster_name
+  disk_size_gb = 5
+
+  // Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  provider_backup_enabled     = true //enable cloud provider snapshots
+  provider_encrypt_ebs_volume = false
+}
+
+resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+  project_id   = mongodbatlas_cluster.cluster_test.project_id
+  cluster_name = mongodbatlas_cluster.cluster_test.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+
+
+  policies {
+  id = mongodbatlas_cluster.cluster_test.snapshot_backup_policy.0.policies.0.id
```

Comment:
No, this belongs to a previous version, this test is to create a cloud backup policies in previous version, then in another folder, will do the import part and verify that it will not appear changes showing that it works